homeassistant/components/shopping_list/__init__.py

```diff
@@ -281,3 +320,23 @@ def websocket_handle_clear(hass, connection, msg):
     hass.data[DOMAIN].async_clear_completed()
     hass.bus.async_fire(EVENT, {"action": "clear"})
     connection.send_message(websocket_api.result_message(msg["id"]))
+
+
+@callback
+def websocket_handle_reorder(hass, connection, msg):
```

Comment:
Please use the modern syntax to define the websocket command.

```suggestion
@websocket_api.websocket_command(
    {
        vol.Required("type"): "shopping_list/items/reorder",
        vol.Required("item_ids"): [str],
    }
)
def websocket_handle_reorder(hass, connection, msg):
```

Then for registration:

```python
from homeassistant.components import websocket_api

websocket_api.async_register_command(hass, websocket_handle_reorder)
```

Reply:
fixed

Reply:
fixed